server/models/services/BrowserService.js

```diff
@@ -0,0 +1,261 @@
+const ModelService = require('./ModelService');
+const { BROWSER_ATTRIBUTES, BROWSER_VERSION_ATTRIBUTES } = require('./helpers');
+const { Sequelize, Browser, BrowserVersion } = require('../');
+const { Op } = Sequelize;
+
+// Section :- association helpers to be included with Models' results
```

Comment:
What are these! `Section :-`? I don't think I've seen them before.

Reply:
Those prefixes should have been removed. Missed them 🙂 - but depending on your IDE's configuration, you can use it to quickly navigate around a file so I use it sometimes as a file grows. It's basically bookmarks most IDEs support.